hibernate-reactive-core/src/test/java/org/hibernate/reactive/schema/SchemaUpdateCockroachDBTestBase.java

```diff
@@ -155,21 +155,17 @@ public void testUpdate(TestContext context) {
 												.setParameter( 1, "asimple" )
 												.getResultList()
 												.thenAccept( list -> {
-													context.assertEquals(
-															"CREATE INDEX i_asimple_avalue_astringvalue ON postgres.public.asimple USING btree (avalue ASC, astringvalue DESC)",
-															list.get( 0 )
+													context.assertTrue( list.contains(
+															"CREATE INDEX i_asimple_avalue_astringvalue ON postgres.public.asimple USING btree (avalue ASC, astringvalue DESC)")
 													);
-													context.assertEquals(
-															"CREATE INDEX i_asimple_avalue_data ON postgres.public.asimple USING btree (avalue DESC, data ASC)",
-															list.get( 1 )
+													context.assertTrue( list.contains(
+															"CREATE INDEX i_asimple_avalue_data ON postgres.public.asimple USING btree (avalue DESC, data ASC)")
 													);
-													context.assertEquals(
-															"CREATE UNIQUE INDEX \"primary\" ON postgres.public.asimple USING btree (id ASC)",
-															list.get( 2 )
+													context.assertTrue( list.contains(
+															"CREATE UNIQUE INDEX asimple_pkey ON postgres.public.asimple USING btree (id ASC)")
 													);
-													context.assertEquals(
-															"CREATE UNIQUE INDEX u_asimple_astringvalue ON postgres.public.asimple USING btree (astringvalue ASC)",
-															list.get( 3 )
+													context.assertTrue( list.contains(
+															"CREATE UNIQUE INDEX u_asimple_astringvalue ON postgres.public.asimple USING btree (astringvalue ASC)")
```

Comment:
The problem with using `assertTrue` is that in case of error you we won't see the difference in the log about what we were expecting to find and what we have found.

It's not a big deal but I would replace this with:
```
.thenAccept( list -> assertThat( list ).containsExactlyInAnyOrder(
		"CREATE INDEX i_asimple_avalue_astringvalue ON postgres.public.asimple USING btree (avalue ASC, astringvalue DESC)",
		"CREATE INDEX i_asimple_avalue_data ON postgres.public.asimple USING btree (avalue DESC, data ASC)",
		"CREATE UNIQUE INDEX asimple_pkey ON postgres.public.asimple USING btree (id ASC)",
		"CREATE UNIQUE INDEX u_asimple_astringvalue ON postgres.public.asimple USING btree (astringvalue ASC)"
) )
```

Reply:
Hadn't used that contains method before.  I'll make the changes.